use-timescale/tigerlake.md

```diff
@@ -0,0 +1,327 @@
+
+# Integrate data lakes with $CLOUD_LONG
+
+$LAKE_LONG enables you to build real-time applications alongside efficient data pipeline management within a single 
+system. $LAKE_LONG unifies the $CLOUD_LONG operational architecture with data lake architectures. 
+
+This experimental release is a native integration enabling synchronization between $HYPERTABLEs and relational tables
+running in $SERVICE_LONGs, and Iceberg tables running in [Amazon S3 Tables][s3-tables] in your AWS account. 
+
+## Prerequisites
+
+<IntegrationPrereqsCloud/>
+
+## Integrate a data lake with your $SERVICE_LONG
+
+To connect a $SERVICE_LONG to your data lake:
+
+<Tabs label="Install TimescaleDB">
+
+<Tab title="AWS Management Console">
+
+<Procedure >
+
+1. **Set the AWS region to host your table bucket**
+   1. In [AWS CloudFormation][cmc], select the current AWS region at the top-right of the page.
+   1. Set it to the Region to create your table bucket in. 
+
+   **This must match the region your $SERVICE_LONG is running in**: if the regions do not match AWS charges you for 
+   cross-region data transfer.
+
+1. **Create your CloudFormation stack**
+   1. Click `Create stack`, then select `With new resources (standard)`.
+   1. In `Amazon S3 URL`, paste the following URL, then click `Next`.
+   
+      ```http request
+      https://tigerlake.s3.us-east-1.amazonaws.com/tigerlake-connect-cloudformation.yaml
+      ```
+   
+   1. In `Specify stack details`, enter the following details, then click `Next`:
+      * `Stack Name`: a name for this CloudFormation stack
+      * `BucketName`: a name for this S3 table bucket
+      * `ProjectID` and `ServiceID`: enter the [connection details][get-project-id] for your $LAKE_LONG $SERVICE_SHORT.
+   1. In `Configure stack options` check `I acknowledge that AWS CloudFormation might create IAM resources`, then 
+      click `Next`.
+   1. In `Review and create`, click `Submit`, then wait for the deployment to complete. 
+      AWS deploys your stack and creates the S3 table bucket and IAM role.
+   1. Click `Outputs`, then copy all four outputs.
+
+1. **Connect your $SERVICE_SHORT to the data lake**
+
+   1. In [$CONSOLE][services-portal], select the $SERVICE_SHORT you want to integrate with AWS S3 Tables, then click 
+      `Connectors`.
+   
+   1. Select the Apache Iceberg connector and supply the:
+      - ARN of the S3Table bucket
+      - ARN of a role with permissions to write to the table bucket   
+
+   Provisioning takes a couple of minutes, during this time the $SERVICE_SHORT is restarted.
+
+</Procedure>
+
+</Tab>
+
+<Tab title="AWS CloudFormation CLI">
+
+<Procedure >
+
+1. **Create your CloudFormation stack** 
+
+   Replace the following values in the command, then run it from the terminal:
+
+   * `StackName`: the name for this CloudFormation stack
+   * `BucketName`: the name of the S3 table bucket to create
+   * `ProjectID`: enter your $SERVICE_LONG [connection details][get-project-id] 
```

Comment:
Project ID and service ID are combined in a single bullet in the first tab, think we can do the same here (or split there as well)